client/src/components/BubblePage.js

```diff
@@ -1,17 +1,29 @@
 import React, { useState, useEffect } from "react";
-import axios from "axios";
+import { axiosWithAuth } from '../utils/axiosWithAuth'
 
 import Bubbles from "./Bubbles";
 import ColorList from "./ColorList";
 
 const BubblePage = () => {
   const [colorList, setColorList] = useState([]);
-  // fetch your colors data from the server when the component mounts
-  // set that data to the colorList state property
+
+  const colors = () => {
+    axiosWithAuth()
+      .get('/api/colors')
+      .then(response => {
+        // console.log(response.data)
+        setColorList(response.data)
+      })
+      .catch(error => console.log(error))
+  }
+
+    useEffect(() => {
+      colors();
+    }, [])
```

Comment:
Creating a function that performs the axios call so you can use it inside your `useEffect()` _and_ pass it down as a prop.

I like it.